src/main/java/com/intuit/graphql/orchestrator/xtext/XtextGraph.java

```diff
@@ -226,6 +233,7 @@ public XtextGraph transform(Consumer<Builder> builderConsumer) {
     private Map<FieldContext, ArgumentsDefinition> resolverArgumentFields = new HashMap<>();
     private Set<DirectiveDefinition> directives = new HashSet<>();
     private Map<String, TypeDefinition> types = new HashMap<>();
+    private Map<String, TypeDefinition> entities = new HashMap<>();
```

Comment:
maybe rename to entitiesByTypename?  so that the variable name self document the key being used.